packer/images.json.pkr.hcl

```diff
@@ -0,0 +1,115 @@
+source "amazon-ebs" "cos" {
+  access_key    = var.aws_access_key
+  ami_name      = "cos-${var.cos_version}-${formatdate("DDMMYYYY", timestamp())}"
+  ami_description = "cos-${var.cos_version}-${formatdate("DDMMYYYY", timestamp())}"
+  instance_type = "t3.small"
+  region       = var.aws_region
+  secret_key   = var.aws_secret_key
+  ssh_password = "cos"
+  ssh_username = "root"
+  source_ami_filter {
+    filters = {
+      name                = "*cos*recovery*"
```

Comment:
Also probably this one too should be customizable, with a default (like this one)